Formula/convco-cli.rb

```diff
@@ -0,0 +1,32 @@
+class ConvcoCli < Formula
```

Comment:
Both the project and the tool is called `convco`, so the formula should probably also be called `convco`.

Reply:
Ok, will rename. I used another rust tool the other day and the formula was `-cli` so I followed the same pattern.